qutebrowser/config/configdata.yml

```diff
@@ -1565,6 +1565,19 @@ url.open_base_url:
   default: false
   desc: Open base URL of the searchengine if a searchengine shortcut is invoked without parameters.
 
+url.open_categories_shown:
+  type:
+    name: List
+    valtype: String
+    none_ok: true
+  default:
+    - searchengines
+    - quickmarks
+    - bookmarks
+    - history
+  desc: Which categories should be shown in the :open dialogue.
```

Comment:
I'd suggest `Which categories to show in the :open dialogue.` for consistency (see https://github.com/qutebrowser/qutebrowser/commit/01b29cd64051d55c73102e8c6062fb743243bbd0).